lib/log.js

```diff
@@ -1,12 +1,61 @@
 'use strict';
-
 const Winston = require('winston');
 
+/** @module Log */
+
 /**
  * Logging provider
  *
  * @global
  */
 const Log = global.Log = new Winston.Logger();
 
-Log.add(Winston.transports.Console, Config.get('log'));
+/*
+ * Ensure that capitalized levels don't crash the colorize routine
+ */
+Winston.addColors({
+  ERROR: 'red',
+  WARN: 'yellow',
+  HELP: 'cyan',
+  DATA: 'grey',
+  INFO: 'green',
+  DEBUG: 'blue',
+  PROMPT: 'grey',
+  VERBOSE: 'cyan',
+  INPUT: 'grey',
+  SILLY: 'magenta'
+});
+
+/**
+ * Log formatting to match levels used by Logback
+ */
+class Logback extends Winston.transports.Console {
+  /**
+   * @constructor
+   * @param  {Object} options
+   */
+  constructor(options) {
+    super(Object.assign({
+      colorize: false,
```

Comment:
When the logger config is set to `json: false, colorize: true` from other configuration, the internal colorize method throws uncaught errors unless there's a matching mapping